### ISSUE TITLE
Inconsistency between tag and pod version

### ISSUE THREAD

**Issue by PatrickNLT** [escoz/QuickDialog#718]
Currently, the pod version is 1.0.2 and the tag is set to 1.1.0. Could you fix that please?

**Comment by nbonamy** [comment]
Seems to be fixed:
$ pod list|grep QuickDialog
  QuickDialog 1.1.0

**Comment by PatrickNLT** [comment]
No, it's not fixed. See https://github.com/escoz/QuickDialog/blob/master/QuickDialog.podspec
